index.js

```diff
@@ -488,7 +488,7 @@ function convertToJsonOnValidate(any) {
   var validate = any._validate;
   any._validate = function () {
     var result = validate.apply(this, arguments);
-    if (!result.error && result.value) {
+    if (!result.error && result.value !== undefined) {
```

Comment:
Perhaps instead of this check should we do `if (!result.error && typeof result.value === 'string')`? Just worried about the `null` case

Reply:
and `false`. Stringifying `false` has no value, but it shouldn't hurt either. I cannot think of why we *shouldn't* stringify these falsey values. We also must account for objects, so if you go the route Chris suggested you would need to stringify anything that was of type `string` or `object`.

Reply:
Yeah - Jacob was right - basically anything that satisfies `x === JSON.parse(JSON.stringify(x))` should go through this path, and `undefined` definitely doesn't work but the rest do - this looks like the right fix